x/staking/simulation/operations.go

```diff
@@ -394,7 +394,7 @@ func SimulateMsgUndelegate(
 		}
 		// if simaccount.PrivKey == nil, delegation address does not exist in accs. Return error
```

Comment:
Could you update this comment and explain the reasoning of why an error isn't returned.

Reply:
Thanks for the review. You are right, the comment there needs to be updated. Sent a fix for review.